lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2021,12 +2142,12 @@ void ModuleEmitter::emitRTLModule(RTLModuleOp module) {
 
     // Emit the type.
     os << portTypeStrings[portIdx];
-    if (portTypeStrings[portIdx].size() < maxTypeWidth)
-      os.indent(maxTypeWidth - portTypeStrings[portIdx].size());
+    if (maxTypeWidth > 0 && portTypeStrings[portIdx].size() < maxTypeWidth + 1)
```

Comment:
As Chris has correctly pointed out in the past, "+1"s look like covering for an off-by-one error elsewhere.

Reply:
But not here.  The issue is that the computed value might be 0, in which case you want 0 spaces.  For anything else, you need to add one space (between the type and the name).

Reply:
that makes sense and the same discontinuity occurs elsewhere in the exportverilog code.